index.bs

```diff
@@ -5632,11 +5632,11 @@ The <dfn export for=commands>emulation.setGeolocationOverride</dfn> command modi
       emulation.GeolocationCoordinates = {
          latitude: float,
          longitude: float,
```

Comment:
For both fields the geolocation spec says that their value is given in degrees. Shall we as well limit to `0.0...360.0)`?

Reply:
Yep, updated (but to valid ranges for these values)